OpencartTest/pom.xml

```diff
@@ -92,19 +92,20 @@
             <artifactId>poi-ooxml</artifactId>
             <version>4.1.2</version>
         </dependency>
-        <!-- https://mvnrepository.com/artifact/io.qameta.allure/allure-testng -->
```

Comment:
good practice for future - explain in PR why you are making changes to project settings